src/main/java/robot/subsystems/drivetrain/pure_pursuit/Path.java

```diff
@@ -1,7 +1,7 @@
 package robot.subsystems.drivetrain.pure_pursuit;
 
 import robot.subsystems.drivetrain.Constants;
-
+import java.math.*;
```

Comment:
Are you sure this import is necessary? As far as I know, you do not need to import the `Math` class, as it is built in. But I may be wrong…
```suggestion

```